src/aks-preview/azext_aks_preview/aks_app_commands/aksappcommands.py

```diff
@@ -0,0 +1,121 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from __future__ import annotations
+import shutil
+import subprocess
+import requests
+import os
+import platform
+from pathlib import Path
+from knack.prompting import prompt_y_n
+
+
+def aks_web_app_init():
+    filePath = _binary_pre_check()
+    if filePath:
+        isRunSuccessful = run_binary(filePath)
+        if isRunSuccessful == 0:
+            cmd_finish()
+            return
+
+    raise ValueError("Binary was NOT executed successfully")
+
+
+# If setup is valid this method returns the correct binary path to execute
+def _binary_pre_check() -> str:
+    print("The DraftV2 setup is in progress ...")
+    draftV2BinaryPath = _find_existing_path(get_potential_paths())
+    if draftV2BinaryPath:
+        return draftV2BinaryPath
+
+    print("DraftV2 binary not found")
+    return download_binary()
+
+
+def _find_existing_path(paths: List[str]) -> str:
+    print("Checking if DraftV2 binary exists ...")
+    operatingSystem = platform.system()
+    # Filename depends on the operating system
+    filename = "draftv2-" + operatingSystem.lower() + "-amd64"
+
+    if not paths:
+        print("List of given DraftV2 paths is empty")
+        return None
+
+    for path in paths:
+        binaryFilePath = path + "/" + filename
+        if os.path.exists(binaryFilePath):
+            print("Existing binary found at: " + binaryFilePath)
+            return binaryFilePath
+    return ""
+
+
+# Returns a list of potential draftV2 binary paths
+def get_potential_paths() -> List[str]:
```

Comment:
📓 `:57:30 F821 undefined name 'List'` - `from typing import List` might be a right solution. for more detail here is help, https://stackoverflow.com/questions/57505071/nameerror-name-list-is-not-defined